src/test/java/bio/terra/service/duos/DuosDaoTest.java

```diff
@@ -66,6 +73,8 @@ public void after() {
   public void testRetrieveFirecloudGroupBeforeInsert() {
     DuosFirecloudGroupModel retrieveBeforeInsert = duosDao.retrieveFirecloudGroupByDuosId(duosId);
     assertNull(retrieveBeforeInsert);
+    List<DuosFirecloudGroupModel> retrieveAllBeforeInsert = duosDao.retrieveFirecloudGroups();
```

Comment:
This seems like a good place to use `var`
```suggestion
    var retrieveAllBeforeInsert = duosDao.retrieveFirecloudGroups();
```